test/SmartEnum.JsonNet.UnitTests/SmartEnumNameConverterTests.cs

```diff
@@ -34,14 +34,14 @@ public class TestClass
             String = TestEnumString.Instance,
         };
 
-        static readonly string JsonString =
-@"{
-  ""Bool"": ""Instance"",
-  ""Int16"": ""Instance"",
-  ""Int32"": ""Instance"",
-  ""Double"": ""Instance"",
-  ""String"": ""Instance""
-}";
+        static readonly string JsonString = JsonConvert.SerializeObject(new
```

Comment:
I was getting a few broken tests due to line ending differences, so I went with this approach.